src/api/bank-accounts/bank-account-requests.md

```diff
@@ -0,0 +1,114 @@
+---
+layout:  default
+grand_parent: API Reference
+parent: Bank Accounts
+title: Bank Account Requests
+permalink: /api/bank-account-requests
+---
+
+# Bank Account Requests
+{:.no_toc}
+
+Bank Account Requests are used to give Centrapay authorization to third party bank accounts via the OAuth 2 flow.
+Upon successful completion of the OAuth 2 flow, Centrapay resources may be privioned from third party resources.
+
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Bank Account Request
+{% h4 Mandatory Fields %}
+
+|          Field           |        Type        |                                        Description                                         |
+| :----------------------- | :----------------- | :----------------------------------------------------------------------------------------- |
+| id                       | String             | The Bank Account Request's unique identifier.                                              |
+| accountId                | String             | The id of the owning Centrapay [Account][].                                                |
+| type                     | String             | The type of Bank Account Request. See [Bank Account Request Types][].                      |
+| status                   | String             | The current status of the Bank Account Request. Supported values are `created` and `done`. |
+| createdAt                | {% dt Timestamp %} | When the Bank Account Request was created.                                                 |
+| createdBy                | {% dt CRN %}       | The User or API Key that created the Bank Account Request.                                 |
+| updatedAt                | {% dt Timestamp %} | When the Bank Account Request was updated.                                                 |
+| updatedBy                | {% dt CRN %}       | The User or API Key that updated the Bank Account Request.                                 |
+| authorizationRedirectUrl | String             | A redirect url to begin the OAuth 2 flow.                                                  |
+
+### Bank Account Request Types
+
+|     Field      |                                   description                                    |
+| :------------- | :------------------------------------------------------------------------------- |
+| quartz-details | Request will initiate an Oauth 2 flow for authorizing access to account details. |
+| quartz-payment | Request will initiate an Oauth 2 flow for authorizing access to create payments. |
+
+## Operations
+
+### Authorize Bank Account Request **EXPERIMENTAL**
+{% reqspec %}
+  POST '/api/bank-account-requests'
+  auth 'api-key'
+  example {
+    body ({
+      accountId: 'uooxSens6ykJaim1Cu1Q55',
+      type: 'quartz-details',
+    })
+  }
+{% endreqspec %}
+
+{% h4 Required Fields %}
```

Comment:
You should join optional and required fields. See https://github.com/centrapay/centrapay-docs/blob/master/src/api/batch/verifone-terminal-status.md?plain=1#L34